### ISSUE TITLE
sqlfluff not running/returning results, version is blank. 

### ISSUE THREAD

**Issue by Snuffy2** [sqlfluff/vscode-sqlfluff#159]
VSCode: 1.93.1
sqlfluff: 3.2.0
vscode-sqlfluff: 3.3.0
Python 3.12.4

sqlfluff is no longer running on the sql files. It did run last week and since then my Python version and vscode-sqlfluff was updated. I don't know if sqlfluff itself updated or not.

The VSCode SQLFluff.log: 
```
--------------------Executing Command--------------------

C:\Users\<name>\scripts\WinPython\WPy64-31241\python-3.12.4.amd64\Scripts\sqlfluff --version

------------------------------------------------------------

Received close event, code 1 signal null
Raw stdout output:

------------------------------------------------------------

sqlfluff version
{
  "major": 0,
  "minor": 0,
  "patch": 0,
  "preRelease": null,
  "build": null
}

------------------------------------------------------------

Reading from file, not stdin

--------------------Executing Command--------------------

C:\Users\<name>\scripts\WinPython\WPy64-31241\python-3.12.4.amd64\Scripts\sqlfluff lint --format json --dialect snowflake --exclude-rules LT09,AM04,CV03,ST06 Script-1.sql

------------------------------------------------------------

Received close event, code 1 signal null
Raw stdout output:

------------------------------------------------------------



------------------------------------------------------------
```

Running sqlfluff from PowerShell:
```
PS C:\Users\<name>\scripts\WinPython\WPy64-31241\python-3.12.4.amd64\Scripts> .\sqlfluff.exe --version
sqlfluff, version 3.2.0

PS C:\Users\<name>\Documents\SQL Scripts> C:\Users\<name>\scripts\WinPython\WPy64-31241\python-3.12.4.amd64\Scripts\sqlfluff lint --format json --dialect snowflake --exclude-rules LT09,AM04,CV03,ST06 Test-1.sql
[{"filepath": "Test-1.sql", "violations": [{"start_line_no": 2, "start_line_pos": 24, "code": "AL05", "description": "Alias 'enc' is never used in SELECT statement.", "name": "aliasing.unused", "warning": false, "fixes": [{"type": "delete", "edit": "", "start_line_no": 2, "start_line_pos": 21, "start_file_pos": 29, "end_line_no": 2, "end_line_pos": 27, "end_file_pos": 35}, {"type": "delete", "edit": "", "start_line_no": 2, "start_line_pos": 20, "start_file_pos": 28, "end_line_no": 2, "end_line_pos": 21, "end_file_pos": 29}], "start_file_pos": 32, "end_line_no": 2, "end_line_pos": 27, "end_file_pos": 35}, {"start_line_no": 3, "start_line_pos": 9, "code": "LT12", "description": "Files must end with a single trailing newline.", "name": "layout.end_of_file", "warning": false, "fixes": 
[{"type": "create_after", "edit": "\n", "start_line_no": 3, "start_line_pos": 10, "start_file_pos": 45, "end_line_no": 3, "end_line_pos": 10, "end_file_pos": 45}], "start_file_pos": 44, "end_line_no": 3, "end_line_pos": 10, "end_file_pos": 45}], "statistics": {"source_chars": 45, "templated_chars": 45, "segments": 39, "raw_segments": 25}, "timings": {"templating": 0.0, "lexing": 0.0, "parsing": 0.014999999999417923, "linting": 0.01599999999962165, "AL01": 0.0, "AL02": 0.0, "AL03": 0.0, "AL04": 0.0, "AL05": 0.0, "AL06": 0.0, "AL07": 0.0, "AL08": 0.0, "AL09": 0.0, "AM01": 0.0, "AM02": 0.0, "AM03": 0.0, "AM05": 0.0, "AM06": 0.0, "AM07": 0.0, "CP01": 0.0, "CP02": 0.0, "CP03": 0.0, "CP04": 0.0, "CP05": 0.0, "CV01": 0.0, "CV02": 0.0, "CV04": 0.0, "CV05": 0.0, "CV06": 0.0, "CV07": 0.0, "CV08": 0.0, "CV09": 0.0, "CV10": 0.0, "CV11": 0.0, "JJ01": 0.0, "LT01": 0.0, "LT02": 0.0, "LT03": 0.0, "LT04": 0.0, "LT05": 0.01599999999962165, "LT06": 0.0, "LT07": 0.0, "LT08": 0.0, "LT10": 0.0, "LT11": 0.0, "LT12": 0.0, "LT13": 0.0, "RF01": 0.0, "RF02": 0.0, "RF03": 0.0, "RF04": 0.0, "RF05": 0.0, "RF06": 0.0, "ST01": 0.0, "ST02": 0.0, "ST03": 0.0, "ST04": 0.0, "ST05": 0.0, "ST07": 0.0, "ST08": 0.0, "ST09": 0.0, "TQ01": 0.0}}]
```

**Comment by RobertOstermann** [comment]
Hmm, thats odd. Seems like it is not executing sqlfluff, maybe something with the `sqlfluff.executablePath`? If you revert to the previous version of the extension that you were on does it start working again?

**Comment by Snuffy2** [comment]
I changed extension version with no effect. 

However, on further testing it has something to do with [WinPython](https://winpython.github.io/). I tried going from WinPython running Python 3.12.4 to 3.12.3 to 3.12.0 all with the same issue as above. I then went to [Portable Python](https://sourceforge.net/projects/portable-python/) running 3.10.5 (the latest version they released) and it is working. I don't think it has to do with the Python version but rather somehow WinPython is different from Portable Python.

Will close this, thanks.
